repositories/patches/rclcpp-16.0.1.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/rclcpp/src/rclcpp/typesupport_helpers.cpp b/rclcpp/src/rclcpp/typesupport_helpers.cpp
```

Comment:
Regarding the patch names: there is no need to have version names therein. If we ever want to support multiple ROS2 distros, that is probably going to be done via different git branches.